scripts/install-custom.sh

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env bash
+
+set -eux
+
+# -----------------------------------------------------------------------------
+# Utils
+
+build_pkgs () {
+    su --preserve-environment \
+        -c 'yes | aurbuild -d juxt -a <(aurqueue *)' rock
+}
+
+install_pkgs () {
+    # We sync pacman to pick up changes to the juxt repo.
+    pacman --noconfirm -Syu "$@"
+}
+
+# -----------------------------------------------------------------------------
+# aurutils
+
+AURUTILS_DIR="/tmp/aurutils"
+
+git clone https://aur.archlinux.org/aurutils.git "$AURUTILS_DIR"
+chmod go+w "$AURUTILS_DIR" # So we can create .SRCINFO files as the rock user.
+cd "$AURUTILS_DIR"
+
+# Install dependencies ourselves because we won't be able to elevate ourselves
+# via sudo when we `makepkg`.
+install_pkgs \
+    jq \
+    pacutils \
+    repose
```

Comment:
`repose` is on the way out and people are moving to `repoctl` I'm told. Something to look out for in the not too distant future.